src/nft/components/profile/list/NFTListingsGrid.tsx

```diff
@@ -44,7 +44,7 @@ export const NFTListingsGrid = ({ selectedMarkets }: { selectedMarkets: ListingM
 
   return (
     <Column>
-      <Row marginTop="20">
+      <Row position="sticky" top="72" paddingY="24" zIndex="1" backgroundColor="backgroundBackdrop">
```

Comment:
I know this is still a draft, but shouldn't we migrate to Styled Components as we tweak different components?

Reply:
if I was adding a new component i would, but realistically this entire file should be rewritten.  Rewriting one line seems little value add here

Reply:
I agree with Mike here since we're already here and this will save time later on

Reply:
nit: Can you use one of the shared style zIndices?